forest/src/main.rs

```diff
@@ -3,15 +3,14 @@
 
 mod cli;
 mod daemon;
-mod logger;
+pub mod logger;
 mod subcommand;
 
 use cli::{cli_error_and_die, Cli};
 use structopt::StructOpt;
 
 #[async_std::main]
 async fn main() {
-    logger::setup_logger();
```

Comment:
You forgot to initialize the logger when we start the daemon.

Reply:
I'm doing it in `forest/src/subcommand.rs` as I need to get the config first

Reply:
Thanks, missed this. Fixed